terraform/telemetry.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# File usage:
+# python3 telemetry.py --session=$SESSION --project_id=$PROJECT_ID--event=$EVENT --version=$VERSION
+
+import sys
+import json
+import hashlib
+import click
+import time
+
+from google.cloud import pubsub_v1
+
+def obfuscate_project_id(project_id):
```

Comment:
nit: add a comment above to explain why do we need this method

Reply:
Added!